process/block/preprocess/transactions.go

```diff
@@ -396,6 +400,14 @@ func (txs *transactions) computeTxsFromMe(body *block.Body) ([]*txcache.WrappedT
 	return allTxs, nil
 }
 
+func (txs *transactions) shouldSkipMiniBlock(miniBlock *block.MiniBlock) bool {
+	shouldSkipMiniBlock := miniBlock.SenderShardID != txs.shardCoordinator.SelfId() ||
```

Comment:
no need to check for shardID.

Reply:
This is in mainchain not in sovereign. This is how the code is right now.